### PR TITLE
Updated to_ lists

### PR DESCRIPTION
**Pull request by RosaWagner**

### CODE REVIEW

File: to_production.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ofl/marmelad # https://github.com/google/fonts/pull/5529
+ofl/notonaskharabic # https://github.com/google/fonts/pull/5009
+ofl/notosansethiopic # https://github.com/google/fonts/pull/5509
+ofl/notosanshanifirohingya # https://github.com/google/fonts/pull/5114
+ofl/notosansmro # https://github.com/google/fonts/pull/5158
+ofl/notosanssymbols2 # https://github.com/google/fonts/pull/5238
+ofl/notosanstifinagh # https://github.com/google/fonts/pull/5604
+ofl/notoserifethiopic # https://github.com/google/fonts/pull/5510
+ofl/reemkufi # https://github.com/google/fonts/pull/5601
+ofl/reemkufi/static # https://github.com/google/fonts/pull/5601
```
Comment (chrissimpkins) [inline to_production.txt:28]: Site of CI fail

**Comment by chrissimpkins** [comment]
This PR led to a CI fail.  It looks like it is due to the `ofl/reemkufi/static` path that was added to the to_production list.

Will comment the failing path out in #5658.  @RosaWagner Mind having a look after the PR is merged? Ty

**Comment by chrissimpkins** [comment]
> This PR led to a CI fail. It looks like it is due to the `ofl/reemkufi/static` path that was added to the to_production list.
> 
> Will comment the failing path out in #5658. @RosaWagner Mind having a look after the PR is merged? Ty

https://github.com/google/fonts/pull/5658/commits/be985f657edf746ad6ac3d7fe397ceb6595190c3

**Comment by RosaWagner** [comment]
Yep I forgot to remove it, we should update the script so it doesn't put this kind of path anymore.

**Comment by chrissimpkins** [comment]
> Yep I forgot to remove it, we should update the script so it doesn't put this kind of path anymore.

Who can do that?

**Comment by RosaWagner** [comment]
@m4rc1e can do, I opened an issue in gftools: https://github.com/googlefonts/gftools/issues/642

**Pull request merged**